Add CloudFormation stack, fix typo

igorlg opened the pull request:
- Added CloudFormation stack to save queries in CloudWatch Logs Insights
- Added CF template/stack description + 1-click deploy button to README
- Fixed typo in README

igorlg committed:
Add CloudFormation stack, fix typo

- Added CloudFormation stack to save queries in CloudWatch Logs Insights
- Added CF template/stack description + 1-click deploy button to README
- Fixed typo in README

igorlg committed:
Fix output URL to CloudWatch Logs Insights

brianz commented on README.md:
```diff
@@ -8,6 +8,10 @@ For an overview of CloudWatch Logs Insights, see
 
+## Save queries using CloudFormation
+
+Queries described below can be persisted in you CloudWatch Logs Insights page using the CloudFormation template in cloudformation.yaml, or by clicking [![Launch CloudFormation template](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=cwlogs-insights-sample-queries&templateURL=https://raw.gi ...
```
small typo: `s/you CloudWatch/your CloudWatch/`

igorlg replied:
Typo fixed! Thank you!

brianz commented on README.md:
```diff
@@ -8,6 +8,10 @@ For an overview of CloudWatch Logs Insights, see
 
+## Save queries using CloudFormation
+
+Queries described below can be persisted in you CloudWatch Logs Insights page using the CloudFormation template in cloudformation.yaml, or by clicking [![Launch CloudFormation template](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=cwlogs-insights-sample-queries&templateURL=https://raw.gi ...
```
Great idea on the quick launch button!

igorlg replied:
Thank you!

brianz commented on cloudformation.yaml:
```diff
@@ -0,0 +1,166 @@
+Parameters:
+  LogGroups:
+    Type: CommaDelimitedList
+    Description: "(Required) List of CloudWatch Log Groups to apply queries to."
```
Let's remove the limitation of applying queries to specific log groups. It's easier to create these queries so that they can be applied to any log group.

I removed this parameter and `LogGroupNames` from the `QueryDefinition`s and it makes this stack much easier to deploy. The queries are also easier to run since you can select any log group in the console.

igorlg replied:
Done. Removed the parameter and references from QueryDefinition. Thanks for pointing that out.

brianz reviewed (changes requested):
Thank you for the PR! Looks good overall...with the removal of `LogGroupNames` as a restriction, it'll be easier to use and deploy.

igorlg committed:
Implement suggested changes

- Fix typo in README.md
- Remove parameter `LogGroups` and it's reference in QueryDefinitions
